jcl/src/java.base/share/classes/openj9/internal/criu/security/CRIUConfigurator.java

```diff
@@ -43,6 +48,8 @@ public final class CRIUConfigurator {
 	/** Tracing for CRIUSEC. */
 	private static final boolean debug = Boolean.parseBoolean(
 			GetPropertyAction.privilegedGetProperty("enable.j9internal.checkpoint.security.api.debug", "false"));
+	private static final Map<String, Set<String>> cachedAlgorithms = new HashMap<>();
```

Comment:
Suggest you use `new TreeMap<>(String.CASE_INSENSITIVE_ORDER)` here to avoid the need to use `toUpperCase()`.

Reply:
Updated the code to use TreeMap.